README.md

```diff
@@ -1,6 +1,21 @@
-# curriculum-cr-training-mobile-menu
+# Portfolio
+ 
+## Built With
 
+- HtML, CSS, JS
+- VSCode
 
-This repo host an initial code for the tasks performed during the Code Reviewers training.
+## Authors
```

Comment:
- [ ] Noticed a typo in HTML let have it all uppercase  and also I suggest  making it `Author` instead of `Authors` since it was just you working on a project to not cause any confusions